GMX-Bots/Large-Open-and-Increase-Positions/package.json

```diff
@@ -21,20 +20,22 @@
     "disable": "forta-agent disable",
     "enable": "forta-agent enable",
     "keyfile": "forta-agent keyfile",
-    "test": "jest --detectOpenHandles",
-    "format": "prettier --write \"src/**/*.ts\""
+    "format": "prettier --write \"src/**/*.ts\"",
+    "test": "jest --detectOpenHandles"
   },
   "dependencies": {
     "forta-agent": "^0.1.6"
   },
   "devDependencies": {
     "@types/jest": "^27.0.1",
     "@types/nodemon": "^1.19.0",
-    "forta-agent-tools": "^2.0.6",
+    "bignumber.js": "^9.0.2",
```

Comment:
why `"bignumber.js"` as a devDependency?, I am pretty sure that if needed it should be a regular dependency

Edit: Yes, I can confirm the above, based on the bot code